panda:
  ros__parameters:
    control_dt: 1000.0 # Hz
    max_joint_speed: 0.1 # rad/s
    joint_controller_name: 'joint_group_position_controller'
    joint_control_topic: 'joint_group_position_controller/commands'
    joint_trajectory_topic: 'joint_trajectory_controller/commands'
    end_effector_target_topic: 'end_effector_target_pose'
    end_effector_pose_topic: 'end_effector_pose'
    model_file: 'description/models/panda/panda.urdf'
    base_frame: 'panda_link0'
    end_effector_frame: 'end_effector_frame'
    finger_joint_tag: 'finger' # tag for the joints to exclude from the IK optimization
    arm_joint_tag: 'panda_joint'
    initial_joint_angles: [0., -0.785, 0., -2.356, 0., 1.571, 0.785, 0., 0.]
    joint_remapping: # necessary because ros messes up the joint states ordering
      panda_joint1:         6
      panda_joint2:         1
      panda_joint3:         8
      panda_joint4:         3
      panda_joint5:         2
      panda_joint6:         4
      panda_joint7:         5
      panda_finger_joint1:  0
      panda_finger_joint2:  7